src/index.html

```diff
@@ -22,6 +22,316 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+          class="header__logo"
```

Comment:
The class 'header__logo' is used both on the <a> and <img> elements. Ensure that this is intentional and that the styles are applied as expected.